tools/changelog-helper.sh

```diff
@@ -112,32 +114,38 @@ group_entries() {
     # Prepend a number to known categories in order to make their sorting position consistent:
     category_order=(
         "$TRANSLATION_ENTRY_TEXT"
-        "GUI"
-        "Accessibility"
-        "Client"
-        "Server"
-        "Recorder"
-        "Performance"
-        "CLI"
-        "Bug Fix"
-        "Windows"
-        "Installer"
-        "Linux"
-        "Mac"
-        "Android"
-        "iOS"
-        "Translation"
-        "Doc"
-        "Website"
-        "Github"
-        "Build"
-        "Autobuild"
-        "Code"
-        "Internal"
+        "$TRANSLATION_ENTRY_TEXT2"
+        "GUI:"
+        "Accessibility:"
+        "Client:"
+        "Client/Server:"
+        "Server:"
+        "Recorder:"
+        "Performance:"
+        "CLI:"
+        "Bug Fix:"
+        "Windows:"
+        "Installer:"
+        "Linux:"
+        "Mac:"
+        "Android:"
+        "iOS:"
+        "Dependencies:"
+        "Translation:"
```

Comment:
Should all the translations be together (i.e. up with `$TRANSLATION_ENTRY_TEXT` and `$TRANSLATION_ENTRY_TEXT2`) - or down here with doc and website?

Reply:
Yes. There are some entries which are sorted by language.

Reply:
I've rearranged them so `Translations:` comes first, then Weblate, then the one starting with `Gui:`, so it groups with the `Gui:` section that follows. It seems that since 3.11.0 only the Weblate category has been used anyway.